ner_v2/detectors/numeral/number/standard_number_detector.py

```diff
@@ -21,18 +24,21 @@ def __init__(self, entity_name, data_directory_path, min_digit, max_digit):
         self.original_date_text = []
         self.entity_name = entity_name
         self.tag = '__' + entity_name + '__'
-        self.min_digit = min_digit
-        self.max_digit = max_digit
 
         self.numbers_word = {}
```

Comment:
Minor: Rename to `numbers_word_map`. Since other two are also have suffix `_map`